community/sway/etc/sway/config.d/99-autostart-applications.conf

```diff
@@ -21,3 +21,8 @@ exec_always {
     '[ -x "$(command -v sworkstyle)" ] && pkill sworkstyle; sworkstyle &> /tmp/sworkstyle.log'
     '[ -x "$(command -v playerctl)" ] && pkill playerctl; playerctl -a metadata --format \'{{status}} {{title}}\' --follow | while read line; do pkill -RTMIN+5 waybar; done'
 }
+
+# https://github.com/Alexays/Waybar/issues/1093#issuecomment-841846291
```

Comment:
Wow, did we even have this issue? I still don't fully get what's wrong after reading through the issue

Reply:
I had on one machine, not the other. I think its a specific combination of pipewire and machine. Don't really know. But as I see it, it is harmless for everybody else, so I though we can share it with everybody. the video in that issue shows the behaviour exactly how it is on my work-machine.